paypal/paypal.php

```diff
@@ -309,7 +309,7 @@ public function onTP_Processpayment($data)
 
 		if (!$verify)
 		{
-			return false;
+			throw new Exception(JText::_('PLG_PAYPAL_PAYMENET_UNABLE_VALIDATE_IPN'));
```

Comment:
Yes. 
The consumer of CPG plugins (eg com_shika) will use try catch to call and we need to do changes in all the plugin for exception i.e in onTP_Processpayment